miner/proposal_builder.go

```diff
@@ -139,6 +141,12 @@ func WithMinGoodAtxPct(pct int) Opt {
 	}
 }
 
+func WithEmitEmptyActiveSet(lid types.LayerID) Opt {
```

Comment:
should be called with the tortoise param?

btw, why is it a tortoise param?

Reply:
i think that this is a part of tortoise, e.g not the core protocol but ballot is a part of tortoise. it is just unfortunate separation